TYPO3.Neos/Classes/TYPO3/Neos/EventLog/Domain/Repository/EventRepository.php

```diff
@@ -59,9 +59,7 @@ public function removeAll()
         $classMetaData = $this->entityManager->getClassMetadata($this->getEntityClassName());
         $connection = $this->entityManager->getConnection();
         $databasePlatform = $connection->getDatabasePlatform();
-        $connection->query('SET FOREIGN_KEY_CHECKS=0');
         $truncateTableQuery = $databasePlatform->getTruncateTableSql($classMetaData->getTableName());
         $connection->executeUpdate($truncateTableQuery);
-        $connection->query('SET FOREIGN_KEY_CHECKS=1');
```

Comment:
Don't get why it is safe to remove this. Does the changes in TYPO3CRIntegrationService prevent errors being thrown?